h2o-app/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
   compile project(":h2o-algos")
   compile project(":h2o-automl")
   compile project(":h2o-ext-target-encoder")
+  compile project(":h2o-k8s")
```

Comment:
This should IMHO be in h2o-assemblies/main instead. Otherwise, the code would be bundled in Hadoop distributions as well. Not the worst thing (it should be inactive even if the Hadoop node was running on k8s) but still - if there is no need to bundle it, I would avoid it.

Reply:
Moved to main assembly. Removed from here.